sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/BlobCheckpointStore/BlobsCheckpointStore.cs

```diff
@@ -493,6 +564,55 @@ async Task wrapper(CancellationToken token)
             return result;
         }
 
+        /// <summary>
+        /// Attempts to read a legacy checkpoint JSON format and extract an offset and a sequence number
+        /// {"PartitionId":"0","Owner":"681d365b-de1b-4288-9733-76294e17daf0","Token":"2d0c4276-827d-4ca4-a345-729caeca3b82","Epoch":386,"Offset":"8591964920","SequenceNumber":960180}
```

Comment:
(extreme) nit:  The JSON is super-helpful, but may be more clear if  as `<remarks>`  and expanded to "normal" form.   For example:

```json
{
    "PartitionId": "0",
    "key": "value"
}
```